DEVELOPMENT.md

```diff
@@ -164,6 +164,26 @@ export EVENT=alloc
 
 For more information on the profiler test, see [the profiler test doc](./data-plane/profiler/README.md).
 
+### Run Filter Benchmarks
+
+If you are building a fitler benchmark or want to benchmark the performance delta caused when changing the filters, you can run:
+
+```shell
+./hack/run.sh benchmark-filter <filter_class_name>
+```
+
+This will run the benchmarks for the class with `<filter_class_name>`. For example, if you want to run all of the Exact Filter Benchmarks, you could run:
```

Comment:
Java class (could be confused w/ a "class annotation" ? 

Perhaps list all options here? (Even we just have that `ExactFilterBenchmark` one)

Reply:
I linked to the existing list of supported classes so that we only have to update one place going forwards